HW_5/task_1.py

```diff
@@ -0,0 +1,12 @@
+# Напишите функцию, которая принимает на вход строку — абсолютный путь до файла. 
+# Функция возвращает кортеж из трёх элементов: путь, имя файла, расширение файла.
+
+
+def separator_string(str_):
+    path, name_ext = str_.rsplit(sep=chr(92), maxsplit=1)
+    return {*name_ext.rsplit(sep=".", maxsplit=1), path}
```

Comment:
Экзотика. Вот еще похожий вариант.
```suggestion
    *str_, file = str_.rsplit(symbol, 1)
    return *str_, *file.split('.')
```